tools/cli/admin.go

```diff
@@ -1043,7 +1052,13 @@ func newAdminFailoverCommands() []cli.Command {
 					Name:  FlagRunIDWithAlias,
 					Usage: "Optional Failover workflow runID, default is latest runID",
 				},
+				cli.BoolFlag{
+					Name: FlagFailoverDrillWithAlias,
+					Usage: "Optional to query failover workflow or failover drill workflow." +
```

Comment:
nit: `query` -> `pause`
Similar for `resume` and `abort` command.

Reply:
updated